app/controllers/guest_rooms_controller.rb

```diff
@@ -0,0 +1,9 @@
+class GuestRoomsController < ApplicationController
+    def create
+        @guest = Guest.find(params[:id])
+        @room = Room.find(params[:room_id])
+        GuestRoom.create(guest_id: @guest.id, room_id: @room.id)
+
+        redirect_to "/guests/#{params[:id]}"
+    end
```

Comment:
Yup, this logic looks good. Could move line 5 into the Model to have the Model, perhaps the Guest model or the GuestRoom model, take care of adding the record.